include/seccomp.h.in

```diff
@@ -336,6 +342,48 @@ struct scmp_arg_cmp {
  */
 #define SCMP_ACT_ALLOW		0x7fff0000U
 
+/*
+ * User notification bits. It's a little unfortunate that we don't export
+ * system.h, so we end up having to define all these structures again.
+ * SECCOMP_RET_USER_NOTIF was added in kernel 5.0. It will not be defined on
+ * older kernels. This version also added the structures below, so let's define
+ * those if the header doesn't have this definiton.
+ */
```

Comment:
We generally don't include commentary like this in the libseccomp header file :)

I think it's both appropriate and a good idea to add a one-liner about user notification first appearing in Linux v5.0 but I would leave it at that.

Reply:
I probably wasn't clear enough about this previously, but this is what I was thinking for a comment:
```
/* User notification bits was added in Linux v5.0 */
```